modules/installation-about-mirror-registry.adoc

```diff
@@ -13,3 +13,12 @@ The mirror registry can be any container registry that supports link:https://doc
 The mirror registry must be reachable by every machine in the clusters that you provision. If the registry is unreachable installation, updating, or normal operations such as workload relocation might fail. For that reason, you must run mirror registries in a highly available way, and the mirror registries must at least match the production availability of your {product-title} clusters.
 
 When you populate a mirror registry with {product-title} images, you can follow two scenarios. If you have a host that can access both the internet and your mirror registry, but not your cluster nodes, you can directly mirror the content from that machine. This process is referred to as _connected mirroring_. If you have no such host, you must mirror the images to a file system and then bring that host or removable media into your restricted environment. This process is referred to as _disconnected mirroring_.
+
+[NOTE]
+====
+For disconnected installations, if configured correctly, even though the image is pulled from the mirrored location: 
+
+* To view the source of pulled images, you must review the `Trying to access` log entry in the CRI-O logs. Other methods to view the image pull source, such as using the `crictl images` command on a node, show the full name of the image.  For example: `quay.io/project/image-name`. 
```

Comment:
“full name” is unclear to me; the mirrored name is also “full”. Consider something like “the originally-requested image name”, “the non-mirrored image name”.